services/QuillLMS/app/controllers/pages_controller.rb

```diff
@@ -19,6 +19,10 @@ class PagesController < ApplicationController
   NUMBER_OF_TEACHERS = "NUMBER_OF_TEACHERS"
   NUMBER_OF_SCHOOLS = "NUMBER_OF_SCHOOLS"
   NUMBER_OF_LOW_INCOME_SCHOOLS = "NUMBER_OF_LOW_INCOME_SCHOOLS"
+
+  # default numbers are current as of 06/03/24
+  DEFAULT_NUMBER_OF_SENTENCES = 2000000000
+  DEFAULT_NUMBER_OF_STUDENTS = 9000000
```

Comment:
I'm wondering if it makes sense to just have these values stored as env variables in Heroku for easier updating?